src/services/fetchdata/fetchdata.jsx

```diff
@@ -12,9 +11,22 @@ export const fetchRegionList = async () => {
   return data;
```

Comment:
I might map through and re-shape your data inside the fetch function instead of all the multi level de-structuring you do throughout the app